libs/wire-api/test/golden/Test/Wire/API/Golden/Generated/Event_conversation.hs

```diff
@@ -35,6 +35,7 @@ import Wire.API.Conversation.Code
 import Wire.API.Conversation.Role
 import Wire.API.Conversation.Typing
 import Wire.API.Event.Conversation
+import Wire.API.Event.LeaveReason
```

Comment:
also add a manual golden test for `ConversationRemoveMembers`?

Reply:
Thanks for spotting this! I added them in 26b485c7f.